Display.cpp

```diff
@@ -0,0 +1,87 @@
+#include "Display.h"
+#include <iostream>
+
+using namespace std;
+
+Display::Display(vector<LogLine>& set) : dataset(set) {}
+
+void Display::printPV(string pid, string vid)
+{
+    for(auto index : dataset)
+    {
+        if(index.getProcessId() == pid && index.getThreadId() == vid)
+        {
+            print(index);//write print final print function
+        }
+    }
+}
+void Display::printTimeRange(string start, string end)
+{
+    for(auto index : dataset)
+    {
+        doParsing(start);
+        map<string,int> uTimemap;
+        uTimemap = index.getTime();
+        if(uTimemap.count("Hour") && uTimemap.count("Minute") && uTimemap.count("Second") && uTimemap.count("Millisecond"))
```

Comment:
it should be .contains() and not .count()

Reply:
done.
introduced in c++20
may be not supported in cpp installed in my pc